tests/integration/test_charm.py

```diff
@@ -4,26 +4,83 @@
 
 """Test glauth charm."""
 
+import asyncio
+
 import pytest
+from helpers import get_glauth_res
 from pytest_operator.plugin import OpsTest
 
-GLAUTH = "glauth"
 SERIES = ["jammy"]
+GLAUTH = "glauth"
+SSSD = "sssd"
+UBUNTU = "ubuntu"
+JINFO = "juju-info"
+LCLIENT = "ldap-client"
```

Comment:
Not a huge issue but I think these variable names should be slightly more descriptive:

```suggestion
JUJU_INFO = "juju-info"
LDAP_CLIENT = "ldap-client"
```

This way as more integrations are added, it is more obvious what these constants map too. Who knows, maybe ldaps will get its own integration endpoint :sweat_smile: